icons/cup-saucer.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M6 4c0-1 2-1 2-2" />
+  <path d="M12 4c0-1 2-1 2-2" />
+  <path d="M18 4c0-1 2-1 2-2" />
+  <path d="M12 18c-3.3 0-6-2.7-6-6V8h12v4c0 3.3-2.7 6-6 6" />
+  <circle cx="20" cy="12" r="2" />
+  <path d="M2 18a4 4 0 0 0 4 4h12a4 4 0 0 0 4-4Z" />
```

Comment:
1. Not very consistent with `mug`/`coffee`, at the very least the handles should look more alike.
2. I prefer the original steam lines to this version, since these look like simple diagonal lines at 100%. We either need to roll back to the original vertical lines, or the waves need to be more pronounced, see suggestion.

![image](https://github.com/lucide-icons/lucide/assets/17746067/078bc7fb-2967-4380-887a-292906597623)

```suggestion
  <path d="M2 18a4 4 0 0 0 4 4h12a4 4 0 0 0 4-4Z" />
  <path d="M6 8h12v6a4 4 0 0 1-4 4h-4a4 4 0 0 1-4-4Z" />
  <path d="M18 8h1a3 3 0 0 1 3 3v0a3 3 0 0 1-3 3h-1" />
  <path d="M6 4a1 1 0 0 1 1-1 1 1 0 0 0 1-1" />
  <path d="M12 4a1 1 0 0 1 1-1 1 1 0 0 0 1-1" />
  <path d="M18 4a1 1 0 0 1 1-1 1 1 0 0 0 1-1" />
```